src/main/java/me/zeroeightsix/kami/module/modules/movement/Jesus.kt

```diff
@@ -25,16 +26,25 @@ import org.kamiblue.event.listener.listener
         category = Module.Category.MOVEMENT
 )
 object Jesus : Module() {
+
+    var dolphinSetting = register(Settings.b("dolphin", false))
```

Comment:
Use `private val` instead since they are not being accessing outside of the class or being modified